docs/add-ins/addin-tutorial.md

```diff
@@ -141,80 +253,63 @@ Locate the line in the manifest that reads `</DesktopFormFactor>`. Above this li
 </ExtensionPoint>
 ```
 
-Now let's look at exactly what that does.
-
-- The `ExtensionPoint` with `xsi:type="MessageComposeCommandSurface"` indicates that we're defining buttons to add to the message compose window.
-
-- By using an `OfficeTab` element with `id="TabDefault"`, we're indicating we want to add our buttons to the default tab on the ribbon.
+### Update resources in the manifest
 
-- The `Group` element defines the grouping for our buttons, with a label set by the `groupLabel` resource.
+The previous code references labels, tooltips, and URLs that you need to define before the manifest will be valid. Specify this information by making the following changes in the `Resources` section of the manifest. 
 
-- The first `Control` element contains an `Action` element with `xsi:type="ShowTaskPane"`, so this button will open a task pane.
+1. Add the following element as a child of the `bt:Urls` element:
 
-- The second `Control` element contains an `Action` element with `xsi:type="ExecuteFunction"`, so this button will invoke a JavaScript function contained in the function file.
-
-Finally we'll update our resources. The code above referenced labels, tool-tips, and URLs that we need to define before our manifest will be valid.
-
-1. Add the following as a child of the `bt:Urls` element:
-
-   ```xml
+    ```xml
     <bt:Url id="insertGistPaneUrl" DefaultValue="https://localhost:3000/msg-compose/insert-gist.html"/>
-   ```
-1. Change the `DefaultValue` attribute of the `bt:String` element with `id="groupLabel"` to `Git the Gist`.
+    ```
+1. Within the collection of child elements under the `bt:ShortStrings` element, locate the child element with `id="groupLabel"`.  Change the `DefaultValue` attribute of the this element to **Git the gist**.
```

Comment:
a couple of things: remove extra space before "Change.." and update to "...attribute of this element..."

Reply:
Fixed; thanks!